### PR TITLE
APP-932 Add CreateLocation to app proto

### PR DESCRIPTION
**Pull request by DTCurrie**
Adds the `CreateLocation` call to the `app.proto` to allow creation of locations via `gRPC`. This is required for https://viam.atlassian.net/browse/APP-861

### CODE REVIEW

File: proto/viam/app/v1/app.proto [hunk 1]
```diff
@@ -124,6 +126,15 @@ message Location {
   google.protobuf.Timestamp created_on = 3;
 }
 
+message CreateLocationRequest {
+  string name = 1;
+  optional string org_id = 2;
```
Comment (AdamMagaluk) [inline proto/viam/app/v1/app.proto:131]: Is there a reason we cannot just make this required? ListLocations has it already.

Comment (AdamMagaluk) [reply proto/viam/app/v1/app.proto:131]: org_id -> organization_id

File: proto/viam/app/v1/app.proto [hunk 2]
```diff
@@ -124,6 +126,15 @@ message Location {
   google.protobuf.Timestamp created_on = 3;
 }
 
+message CreateLocationRequest {
+  string name = 1;
```
Comment (AdamMagaluk) [inline proto/viam/app/v1/app.proto:130]: nit: Swap the order of name and organization.

File: proto/viam/app/v1/app.proto [hunk 3]
```diff
@@ -124,6 +126,15 @@ message Location {
...
+  optional string org_id = 2;
+}
+
+message CreateLocationResponse {
+  string id = 1;
```
Comment (AdamMagaluk) [inline proto/viam/app/v1/app.proto:135]: Let's just return the Location created.

**Review by AdamMagaluk — CHANGES_REQUESTED** [review]
Minor changes.

**Review by AdamMagaluk — APPROVED** [review]

**Comment by DTCurrie** [comment]
@edaniels I ran `make dist/buf` and am not seeing any changes on my machine.

**Comment by DTCurrie** [comment]
> @edaniels @stevebriskin I ran `make dist/buf` and am not seeing any changes on my machine.

Just needed to run `make setup` again.

**Pull request merged**